### PR TITLE
youlean-loudness-meter 2.2.4 (new cask)

### PR DESCRIPTION
**Pull request by rlpvin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/y/youlean-loudness-meter-2.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  url "https://cdn.youlean.co/wp-content/uploads/2023/03/Youlean-Loudness-Meter-2-V#{version}-macOS.zip"
+  name "Youlean Loudness Meter 2"
+  desc "Loudness meter plugin and standalone app"
+  homepage "https://youlean.co/youlean-loudness-meter/"
+
+  livecheck do
+    url "https://youlean.co/download/"
+    strategy :page_match
+    regex(%r{href=.*?/Youlean-Loudness-Meter-2-V(\d+(?:\.\d+)+)-macOS.zip}i)
```
Comment (razvanazamfirei) [inline Casks/y/youlean-loudness-meter-2.rb:13]:
```suggestion
    regex(%r{href=.*?/(\w+)/(\w+)/youlean[_.-]loudness[_.-]meter-#{version.major}-v?(\d+(?:\.\d+)+)[_.-]macOS\.zip}i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[2]},#{match[0]},#{match[1]}" }
    end
```

File: Casks/y/youlean-loudness-meter-2.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  uninstall pkgutil: [
+    "com.Youlean.aax.pkg.YouleanLoudnessMeter2",
+    "com.Youlean.au.pkg.YouleanLoudnessMeter2",
+    "com.Youlean.standalone.pkg.YouleanLoudnessMeter2",
+    "com.Youlean.vst.pkg.YouleanLoudnessMeter2",
+    "com.Youlean.vst3.pkg.YouleanLoudnessMeter2",
+  ]
+
+  zap trash: "~/Library/Application Support/Youlean/Youlean Loudness Meter 2/*"
```
Comment (razvanazamfirei) [inline Casks/y/youlean-loudness-meter-2.rb:26]:
```suggestion
  zap trash: "~/Library/Application Support/Youlean/Youlean Loudness Meter #{version.major}"
```

Comment (razvanazamfirei) [reply Casks/y/youlean-loudness-meter-2.rb:26]: Is this the only zap needed?

File: Casks/y/youlean-loudness-meter-2.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+cask "youlean-loudness-meter-2" do
+  version "2.4.4"
+  sha256 "7fcc90ad99d2b2bdba60d5a00cd128bb4efaf3fa2982e965026f1c21841d4132"
+
+  url "https://cdn.youlean.co/wp-content/uploads/2023/03/Youlean-Loudness-Meter-2-V#{version}-macOS.zip"
```
[the remark this answers is not in the document]
Comment (razvanazamfirei) [reply Casks/y/youlean-loudness-meter-2.rb:5]:
```suggestion
  url "https://cdn.youlean.co/wp-content/uploads/#{version.csv.second}/#{version.csv.third}/Youlean-Loudness-Meter-#{version.major}-V#{version.csv.first}-macOS.zip"
```

File: Casks/y/youlean-loudness-meter-2.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+cask "youlean-loudness-meter-2" do
+  version "2.4.4"
```
Comment (razvanazamfirei) [inline Casks/y/youlean-loudness-meter-2.rb:2]:
```suggestion
  version "2.4.4,2023,03"
```

**Review by razvanazamfirei — CHANGES_REQUESTED** [review]

File: Casks/y/youlean-loudness-meter-2.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (razvanazamfirei) [reply Casks/y/youlean-loudness-meter-2.rb]: @rlpvin, no worries! I made the changes and committed them! Thanks so much for your first contribution!

**Review by razvanazamfirei — APPROVED** [review]

File: Casks/y/youlean-loudness-meter.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  livecheck do
+    url "https://youlean.co/download/"
+    regex(%r{href=.*?/(\w+)/(\w+)/youlean[._-]loudness[._-]meter[._-]#{version.major}[._-]v?(\d+(?:\.\d+)+)[._-]macOS\.zip}i)
+    strategy :page_match do |page, regex|
+      page.scan(regex).map { |match| "#{match[2]},#{match[0]},#{match[1]}" }
+    end
+  end
+
+  auto_updates true
+  depends_on macos: ">= :mavericks"
```
Comment (bevanjkay) [inline Casks/y/youlean-loudness-meter.rb:19]: CI doesn't know mavericks anymore so this can and should be removed.

**Review by bevanjkay — DISMISSED** [review]

File: Casks/y/youlean-loudness-meter.rb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  sha256 "7fcc90ad99d2b2bdba60d5a00cd128bb4efaf3fa2982e965026f1c21841d4132"
+
+  url "https://cdn.youlean.co/wp-content/uploads/#{version.csv.second}/#{version.csv.third}/Youlean-Loudness-Meter-#{version.major}-V#{version.csv.first}-macOS.zip"
+  name "Youlean Loudness Meter"
+  desc "Loudness meter"
+  homepage "https://youlean.co/youlean-loudness-meter/"
+
+  livecheck do
+    url "https://youlean.co/download/"
+    regex(%r{href=.*?/(\w+)/(\w+)/youlean[._-]loudness[._-]meter[._-]#{version.major}[._-]v?(\d+(?:\.\d+)+)[._-]macOS\.zip}i)
```
Comment (krehel) [inline Casks/y/youlean-loudness-meter.rb:12]: Won't this break when #{version.major} is incremented to 3 on the download site but the Cask will be at version 2?

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @rlpvin.

**Pull request merged**